mingw-w64-vsg/PKGBUILD

```
@@ -0,0 +1,75 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.pro>
+
+_fullname=VulkanSceneGraph
+_realname=vsg
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
+         "${MINGW_PACKAGE_PREFIX}-${_realname}-docs")
+pkgver=1.1.3
+pkgrel=1
+pkgdesc="Vulkan & C++17 based Scene Graph Project (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='http://www.vulkanscenegraph.org/'
+msys2_repository_url='https://github.com/vsg-dev/VulkanSceneGraph'
+license=('spdx:MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-vulkan-loader"
+         "${MINGW_PACKAGE_PREFIX}-libwinpthread-git")
```

```suggestion
depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
         "${MINGW_PACKAGE_PREFIX}-vulkan-loader"
         "${MINGW_PACKAGE_PREFIX}-libwinpthread-git")
```
see runtime dependencies in CI